charts/ceph-csi-rbd/values.yaml

```diff
@@ -203,6 +203,9 @@ provisioner:
       pullPolicy: IfNotPresent
     resources: {}
 
+  # set metadata on volume
+  setmetadata: true
```

Comment:
disable it by default until the ceph issue is fixed?

Reply:
We should comment out the test cases then?

Reply:
Ah! Noticed this now :-)
Still need to address it.

Reply:
Done now.